manual/src/cmds/intf-c.etex

```diff
@@ -464,7 +464,7 @@ floating-point numbers.}
 In earlier versions of OCaml, it was possible to use
 word-aligned pointers to addresses outside the heap as OCaml values,
 just by casting the pointer to type "value".  Starting with OCaml
-4.11, this usage is deprecated and will stop being supported in OCaml 5.00.
+4.11, this usage is deprecated and will stop being supported in OCaml 5.0.
```

Comment:
You can be more assertive:

This usage is no longer supported since OCaml 5.0.